BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,195 @@
+# Manage BigCommerce connection
+
+*Actindo* features a special installation app in the *BigCommerce App Store*. The *Actindo Core1 DOP connector* app setups and authorizes a connection automatically. 
+Note, if possible, you should not delete the app in *BigCommerce* after creating the connection. Although the deleting will not delete the connection in *Omni-Channel* itself, but it will invalidate the access token you have created during connecting the app. If the app was deleted by mistake, either reconnect the app or enter a valid access token manually to enable the connection again.
```

Comment:
Wenn die Kunden das sowieso nicht machen sollten, wollen wir es erwähnen? Ich finde, es klingt nicht "wichtig" genug ("Note, if possible,..."). Vielleicht ein Info-Box (oder sogar Caution) mit der Info, was sie tun sollten, FALLS es aus Versehen gelöscht wird? 
Umformulierungsvorschlag: Although deleting the connection the app will not delete the connection..., it will invalidate the access token... (Sonst gibt es zwei Konzessive-Sätze, finde ich: Although ..., aber...)